optuna/storages/_rdb/models.py

```diff
@@ -87,6 +86,50 @@ def all(cls, session: orm.Session) -> List["StudyModel"]:
         return session.query(cls).all()
 
 
+class StudyDirectionModel(BaseModel):
+    __tablename__ = "study_direction"
+    __table_args__: Any = (UniqueConstraint("study_id", "objective_id"),)
+    study_direction_id = Column(Integer, primary_key=True)
+    direction = Column(Enum(StudyDirection), nullable=False)
+    study_id = Column(Integer, ForeignKey("studies.study_id"))
+    objective_id = Column(Integer)
```

Comment:
Looks like a primary key at first glance with the id suffix. Do you think that just naming it "objective" is too obfuscated? It could be less confusing.